src/main/resources/ERCode/ERCodeEditSheet.xml

```diff
@@ -259,7 +264,7 @@ $(function(){
     <name>ERCode.ERCodeEditSheet</name>
     <number>0</number>
     <className>XWiki.StyleSheetExtension</className>
-    <guid>3831270f-70f4-401b-a0e1-9f14d856f61e</guid>
+    <guid>ab769336-f2af-47f3-a52f-d635272ddb07</guid>
```

Comment:
This is another thing that you should generally avoid committing. Same reasons as the 'date' elements at the beginning of the xml document.